### PR TITLE
kernelCTF: add CVE-2023-3390_lts_cos_mitigation

### PR DESCRIPTION
**Pull request by c0m0r1**

### CODE REVIEW

File: pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/metadata.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+    "$schema":"https://google.github.io/security-research/kernelctf/metadata.schema.v1.json",
```
Comment (koczkatamas) [inline pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/metadata.json:2]: Switch to the latest schema (v2).
```suggestion
    "$schema":"https://google.github.io/security-research/kernelctf/metadata.schema.v2.json",
```

File: pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/metadata.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    "submission_id":[
```
Comment (koczkatamas) [inline pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/metadata.json:3]: In v2, the property was renamed:
```suggestion
    "submission_ids":[
```

File: pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/metadata.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+       "exp48",
+       "exp49",
+       "exp50"
+    ],
+    "vulnerability":{
+       "patch_commit":"https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?id=1240eb93f0616b21c675416516ff3d74798fdc97",
+       "cve":"CVE-2023-3390",
+       "affected_versions":[
+          "v.3.1-rc1 - v6.4-rc1"
```
Comment (koczkatamas) [inline pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/metadata.json:12]: Fix version format:
```suggestion
          "3.1-rc1 - 6.4-rc1"
```

File: pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/exploit/cos-105-17412-101.17/Makefile
Comment (koczkatamas) [inline pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/exploit/cos-105-17412-101.17/Makefile:1]: Please rename the directory `cos-105-17412-101.17` to `cos-105-17412.101.17`.

File: pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/exploit/refined/Makefile
Comment (koczkatamas) [inline pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/exploit/refined/Makefile:1]: Please rename the directory `refined` to `extra-refined`.

File: pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/exploit/exploit.tar.gz
Comment (koczkatamas) [inline pocs/linux/kernelctf/CVE-2023-3390_lts_cos_mitigation/exploit/exploit.tar.gz:1]: This file is not required if it contains the same files as in the exploit directory, please remove it.

**Review by koczkatamas — CHANGES_REQUESTED** [review]

**Comment by koczkatamas** [comment]
Hey!

Sorry for the late response. We created a Github Actions job to verify the submission PRs.

The current test run mostly failed because of the PR's file structure issues, see my suggestions above how to resolve these. You can see the current test run results here: https://github.com/google/security-research/actions/runs/5964212828

After you fix these issues the Github Action will run again, and it will test the exploit compilation and exploit reproduction too.

The compilation failed for us with the error `configure: error: Package requirements (libmnl >= 1.0.4) were not met: No package 'libmnl' found`, but you will see too from the GHA logs after you fix the structure issues and rerun the GHA job.

The exploit reproduction worked for `cos` but failed with `mitigation` and `lts`, because they wait for `[+] type any number to go!`, please remove the requirement of user input from the exploits.

If the reproduction fails for some other reason too, then please take a look why it fails. The reproduction system is a bit different than the live one (it runs the exploit directly from `/init` and there is no nsjail). You can also give feedback how we should modify our reproduction procedure.

So feel free to modify the PR and the Github Action will run again and you will see the new verification results.

Thank you for your submission and participating in kernelCTF!

**Comment by c0m0r1** [comment]
Hello. 
We fixed all the mentioned issues. However, It seems to require CLA from @koczkatamas, since you're the co-author of commits for fixing metadata. 
Should I wait for the CLA signing from you or revert that commits?

**Comment by koczkatamas** [comment]
Hey!

Thanks for the changes!

I solved the CLA issue.

Sadly your exploit still does not build, so the reproduction uses the old exploit which still contains the user interaction, so the repro also fails.

I also changed the way how the Github Actions job work and hopefully you will be trigger a re-check automatically in the future when you make changes to the PR.

I now re-run the checks manually and these are the current results: https://github.com/google/security-research/actions/runs/5969023328

**Review by koczkatamas — APPROVED** [review]

**Pull request merged**